pkg/kv/kvserver/replica_raft.go

```diff
@@ -1511,10 +1511,17 @@ func (m lastUpdateTimesMap) updateOnBecomeLeader(descs []roachpb.ReplicaDescript
 	}
 }
 
-// isFollowerActive returns whether the specified follower has made
-// communication with the leader in the last MaxQuotaReplicaLivenessDuration.
+// isFollowerActive is like isFollowerActiveSince, with a default threshold.
```

Comment:
I think it might clarify things a little bit if this always required the parameter to be passed in.

Reply:
done